Assets/Level/Scenes/Scene_1.unity

```diff
@@ -123,6 +123,240 @@ NavMeshSettings:
     debug:
       m_Flags: 0
   m_NavMeshData: {fileID: 0}
+--- !u!1 &9778842
+GameObject:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  serializedVersion: 6
+  m_Component:
+  - component: {fileID: 9778847}
+  - component: {fileID: 9778846}
+  - component: {fileID: 9778845}
+  - component: {fileID: 9778844}
+  - component: {fileID: 9778848}
+  m_Layer: 0
+  m_Name: Cube
```

Comment:
This `Cube` component represents the `PlayerSpawner`, right? I would rename it as `PlayerSpawner`, change the script name to  `PlayerSpawnerController.cs`, and put it as a prefab. 

In this way we have all the prefabs that can be used as "lego" blocks for the future levels.